editor/src/com/talosvfx/talos/editor/widgets/propertyWidgets/PropertyWidget.java

```diff
@@ -29,6 +30,8 @@
 	private Supplier<T> supplier;
 	private ValueChanged<T> valueChanged;
 
+	private Object parent;
```

Comment:
the connection between property widget and AComponent is lost once the widget is created. Any ideas for better system?

Reply:
Dont get this? Why is it lost?

Reply:
property widget is connected to the actual fields with reflection. So it doesn't know whose fields its changing. I had to keep the parent here and pass it to the change event, in order to send it to socket

Reply:
ah, thats fine